src/build.py

```diff
@@ -1514,7 +1514,7 @@ def TestBare():
   for opt in BARE_TEST_OPT_FLAGS:
     ExecuteLLVMTorture(
         name='d8-lld',
-        runner=Executable(os.path.join(INSTALL_BIN, 'd8')),
+        runner=NODE_BIN,
```

Comment:
Strictly speaking we don't need to run the bare tests in node, just asm and emwasm. Although it's maybe better to run them all in the same shell? I have no preference here.